.github/workflows/ci.yaml

```diff
@@ -227,6 +281,59 @@ jobs:
           . venv\\Scripts\\activate
           pytest tests/
 
+  tests-windows-alpha:
+    name: tests / run / ${{ matrix.python-version }} / Windows
+    runs-on: windows-latest
```

Comment:
Do we want / need to test Windows? For almost everything the `Linux` test should be enough. Once merged we'll test Windows anyway.